### ISSUE TITLE
Hide Templates Feature from any New Accounts

### ISSUE THREAD

**Issue by influx-product-manager** [influxdata/ui#6520]
## User Story

<p></p>

As an organization with new business objectives, I want to prohibit any sign\-ups after Jan 31 from seeing the templates submenu item so that they don't start using a feature that has no additional development planned\.  I want to feature flag control this in case I need to make any exceptions\.

<p></p>

## Acceptance Criteria

<p></p>

- A feature flag will be created to control whether or not the Templates feature is shown in the UI

- If the sign\-up date for the user is after Jan 31st and the feature flag is false, then the Templates submenu item will not be shown

- If the feature flag is true, or the sign\-up date is before Jan 31st, then the Templates  sub\-menu item will be shown

- These changes will only apply to IOx Enabled Orgs\.  For non\-IOx orgs, this hiding should not occur\.

<p></p>

<p></p>

**Comment by influx-product-manager** [comment]
Link to feature: https://influxdata.productboard.com/feature-board/planning/features/17572941

**Comment by garylfowler** [comment]
Note:  Automation Team may have already done some work on this item - so we should check with them before starting.
